params/extras/config.go

```diff
@@ -38,7 +39,13 @@ var (
 	}
 
 	TestChainConfig = &ChainConfig{
-		AvalancheContext:   AvalancheContext{SnowCtx: utils.TestSnowContext()},
+		AvalancheContext: AvalancheContext{
```

Comment:
If someone with additional context could explain why this is necessary here, but was able to be removed in https://github.com/ava-labs/coreth/pull/950/files#r2080300565, that would be appreciated. I think I can also modify so they're set during VM init

Reply:
I think the only difference is in Verify() where we pass c.SnowCtx.NetworkUpgrades. I think we should explicitly handle that in tests.

Reply:
Can you expand upon what you mean by this - I see now why we do need the context as the `verifyNetworkUpgrades` method needs to compare configured timestamps against the expected defaults from the Avalanche config.